### ISSUE TITLE
414 Request-URI Too Large

### ISSUE THREAD

**Issue by wwuck** [SoylentNews/rehash#484]
I tried to click "Quote This" on the following comment:

https://soylentnews.org/comments.pl?noupdate=1&sid=37758&page=1&cid=1000907#commentwrap

and I was presented with the server error "414 Request-URI Too Large".

<img width="994" alt="Untitled" src="https://user-images.githubusercontent.com/301402/83353020-fc336200-a392-11ea-8cb4-59d1e53ba373.png">

There is also another UI bug here, in that when I clicked the browser "Back" button from the 414 error page, it returned to the comment page but had appended a second "Quote This" button to the end of the comment.  Clicking the browser "Forward" and then "Back" buttons again resulted in the comment page showing again but this time with three "Quote This" buttons at the end of the comment.  Doing this more times just ends up with more "Quote This" buttons appended to the comment.

<img width="1409" alt="Screen Shot 2020-05-31 at 22 57 40" src="https://user-images.githubusercontent.com/301402/83352914-3fd99c00-a392-11ea-8364-f644315436f7.png">

**Comment by marty-b** [comment]
As far as I can tell, a "Quote This" button is not provided by SoylentNews.com.

I opened your link in my browser both as an Admin user and separately (in Private Browsing mode) as an Anonymous Coward and saw no "Quote This" button appear.

I have a local copy of the source tree for SoylentNews, so I went to the root of that tree and issued this command to perform a recursive, case-insensitive search through all of the site's files:
` grep -iIR "quote this" .`
This, too, came up with no match.

Are you, by chance, using takyon's "Soylent Upgrade" (which is linked to in the comment you were trying to reply to). That link points to:
  [https://soylentnews.org/~takyon/journal/2712](url)
Because of all the testing that I do for the site, I have chosen to not install that "upgrade" so as to (as much as reasonably possible) use a plain vanilla version of the UI.

In my case, the only button I saw on the comment you reference is "Reply to This".

The "Reply to This" button appeared to work for me as expected.

Hope that helps!
